README.md

```diff
@@ -117,7 +168,13 @@ aws serverlessrepo put-application-policy \
 
 ## Install Lambda into AWS
 
+When using AWS Organizations, the lambda should be deployed once in the master
```

Comment:
good point, this is poorly worded. technically using the primary organization account would be perfectly reasonable to get org-wide totals. but since it was adapted for the needs of STRIDES, we need to run it in each account because we don't have access to their primary account.